examples/files.js

```diff
@@ -132,6 +130,12 @@ var files = {
 		"webgl_loader_vrm",
 		"webgl_loader_vrml",
 		"webgl_loader_vtk",
+		"webgl_loader_worker_gltf",
+		"webgl_loader_worker_pcd",
```

Comment:
I do not vote to add a PCD example. A single `glTF` example is sufficient from my point of view. We should keep in mind that we all have to maintain this code. Please keep the changes lean.